parsec/api/protocol/organization.py

```diff
@@ -45,6 +45,7 @@ class OrganizationBootstrapReqSchema(BaseReqSchema):
     device_certificate = fields.Bytes(required=True)
     redacted_user_certificate = fields.Bytes(required=True)
     redacted_device_certificate = fields.Bytes(required=True)
+    sequester_verify_key_certificate = fields.Bytes(required=False, allow_none=True, missing=None)
```

Comment:
Should add a comment `Added in API version 1.4 (Parsec 2.11.0)`

Reply:
And of course we should modify `API_V1_VERSION` field in parsec/api/version.py

Reply:
What is the step to bump the api version ? 
Shall i just bump the API Version in `version.py`, or there is something else ? (like update some api compatibility tests)

Reply:
Currently we don't have a clean way to do that :'(  I was supposed to document an issue about that, but it's still "to be announced"®

So it's just bump the number in `version.py`

Reply:
(the new schema implementation in Rust should make things much easier to handle version bump though ^^)